modules/aws/lb/main.tf

```diff
@@ -0,0 +1,301 @@
+###########################
+# Provider Configuration
+###########################
+terraform {
+  required_version = ">= 1.0.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.0.0"
+    }
+  }
+}
+
+###########################
+# Locals
+###########################
+
+locals {
+  is_application = var.load_balancer_type == "application"
+  is_network     = var.load_balancer_type == "network"
+}
+
+###########################################################
+# Load Balancer
+###########################################################
+
+resource "aws_lb" "load_balancer" {
+  # Common settings
+  name                       = var.name
```

Comment:
Add `client_keep_alive`

Reply:
added client keep alive with alb logic sicne it is alb specific.